pkg/operator/genericoperatorclient/dynamic_operator_client_test.go

```diff
@@ -33,7 +33,6 @@ func TestSetOperatorSpecFromUnstructured(t *testing.T) {
 					"non-standard-field":         "value",
 					"logLevel":                   "Trace",
 					"managementState":            "",
-					"operatorLogLevel":           "",
```

Comment:
these changes look unrelated. If necessary, put into extra commit and describe why youneed them.

Reply:
It is related and a must, the test fails without this change.
The new version of 'api' changes the behavior of the response and that what is expected from now.

Reply:
Then move it into its own commit.

Also please don't mix vendor bump with other changes.

Reply:
done

Reply:
I reorganized the commits in this PR